core/src/main/java/org/lionweb/lioncore/java/metamodel/Metamodel.java

```diff
@@ -35,6 +35,17 @@ public Metamodel setName(String name) {
         return this;
     }
 
+    public Metamodel setVersion(@Nullable Integer version) {
```

Comment:
Didn't we decide to make a Metamodel's version a String?

Reply:
You are right!